pkg/logs/watch_build.go

```diff
@@ -0,0 +1,24 @@
+package logs
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+
+	"github.com/pivotal/kpack/pkg/client/clientset/versioned"
+)
+
+type watchBuild struct {
+	buildName   string
+	kpackClient versioned.Interface
+	namespace   string
+	context     context.Context
+}
+
+func (l *watchBuild) Watch(options v1.ListOptions) (watch.Interface, error) {
```

Comment:
It is disappointing this interface was not updated to accept a context

Reply:
yeah :(